docs/develop/go/message-passing.mdx

```diff
@@ -346,10 +346,24 @@ You must provide the Workflow Id, but specifying a Run Id is optional.
 If you supply only the Workflow Id (and provide an empty string as the Run Id param), the running Workflow Execution receives the Update.
 
 You must provide a `WaitForStage` when calling `UpdateWorkflow()`.
-This parameter controls what stage the update must reach before a handle is returned to the caller. If `WaitForStage` is set to `WorkflowUpdateStageCompleted` the handle is returned after the update completes; if `WaitForStage` is set to `WorkflowUpdateStageAccepted` the handle is returned after the Update is accepted (i.e. after the validator has run, if there is a validator).
+This parameter controls the stage the update must reach before returning a handle to the caller:
+
+- If `WaitForStage` is set to `WorkflowUpdateStageCompleted`, the handle is returned after the Update completes.
+- If `WaitForStage` is set to `WorkflowUpdateStageAccepted`, the handle is returned after the Update is accepted and the Validator is run, if it's defined.
+
+Note: You cannot send Updates to other Workflow Executions.
+
+The following sample demonstrates how to create a context with timeout for sending an `Update` to the Workflow:
+
+If `WaitForStage` is set to `WorkflowUpdateStageCompleted` the handle is returned after the update completes; if `WaitForStage` is set to `WorkflowUpdateStageAccepted` the handle is returned after the Update is accepted (i.e. after the validator has run, if there is a validator).
+
+Also note that you can't send Updates to other Workflow Executions.
```

Comment:
I believe Erica added these to align Go with the other SDKs; as I can see similar copy over there.